package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "wagtail",
+  "version": "1.0.0",
```

Comment:
Speaking of not distributing wagtail over npm, maybe we could add `"private": true` to the package.json as well? It's easy enough to accidentally run `npm publish` in the wrong terminal tab.